specs/ExtendedProcessFailedEventArgs.md

```diff
@@ -0,0 +1,85 @@
+
+Title
+===
+
+# Background
+Code integrity is a feature of Windows that verifies the authenticity and integrity of the code that runs on the system. It helps protect it from malware, tampering, and unauthorized changes. Code integrity checks the digital signatures of the files that are loaded into memory, and prevents any file that does not have a valid signature from running in WebView2. We are extending ProcessFailedEventArgs with BlockedFile property which caused webview2 process to exit with code STATUS_INVALID_IMAGE_HASH.
+
+# Examples
+
+    ```c#
+        /// This is an event handler for our CoreWebView2's ProcessFailedEvent
+        private void CoreWebView2_ProcessFailed(object sender, CoreWebView2ProcessFailedEventArgs e)
+        {
+            if (e.ExitCode == -1073740760 /*STATUS_INVALID_IMAGE_HASH*/)
+            {
+                SendTelemetry(e.BlockedFile);
```

Comment:
Would be nice to add a comment here (and in the corresponding spot in the other sample code) that explains what its doing. eg:
```suggestion
                // If the renderer process crashed because of STATUS_INVALID_IMAGE_HASH, then we
                // want to log to our app's telemetry the name of the DLL that caused the issue.
                SendTelemetry(e.BlockedFile);
```